README.md

```diff
@@ -183,7 +103,17 @@ import curry from 'crocks/helpers/curry'
 import isNumber from 'crocks/predicates/isNumber'
 import liftA2 from 'crocks/helpers/liftA2'
 import safe from 'crocks/Maybe/safe'
+```
 
+### Example
+
+Documentation references: [and](https://crocks.dev/docs/functions/logic-functions.html#and),
```

Comment:
The references should use the reference bracket notation and have the link moved down below.

Reply:
Understood. Would you like me to convert the existing links (besides the badges at the top), that do not follow this pattern?

Reply:
Sure if you wanna.
Unless you feel it will add too much noise to the PR. In that case we can do a PR later, focused on that specifically. Your call totally on that.